src/api/ApiIndex.vue

```diff
@@ -204,6 +226,10 @@ h3 {
   .header {
     display: block;
   }
+
+  .api-filter {
+    justify-content: flex-start;
```

Comment:
I'm wondering whether we could replace the `justify-content: space-between` in the earlier rule with `flex-start`? From what I can tell it would look exactly the same, but without needing this extra tweak in the media query?

Reply:
Yep, you are right. Will fix it. One more idea - is it OK to change the input type to "search" (instead of default "text) as we get the "x" (to cancel search) for free?